orchestra/utils/task_lifecycle.py

```diff
@@ -488,7 +489,8 @@ def tasks_assigned_to_worker(worker):
         .exclude(
             task__status=Task.Status.COMPLETE)
         .exclude(
-            task__project__status=Project.Status.PAUSED)
+            task__project__status__in=[Project.Status.PAUSED,
+                                       Project.Status.COMPLETED])
```

Comment:
will this result in us showing completed tasks in the pending queue on the dashboard?

more broadly, QA the dashboard on `main` and on this branch and make sure the tasks that show up there are sane after your new completion logic

Reply:
It will not show completed tasks/project because we are excluding them.

Reply:
I did the QA and everything looks sane.

Reply:
The QA was really helpful for active/completed. I still have a lingering question: "will this result in us showing completed tasks in the pending queue on the dashboard?"

Can you trace where `inactive_task_assignments` is used? I wonder why we had to exclude completed projects from it.

As additional QA, you can run the same exercise from your screenshot above but for pending/paused tasks and see if completing the project changes their behavior.

Reply:
`inactive_task_assignments` is used to create `inactive_review_task_assignments` and `inactive_processing_task_assignments` which maintain the list of tasks with [pending states](https://github.com/b12io/orchestra/blob/6b2d3b652c69f13cf4431da786c0d0d8156bf98b/orchestra/frontend/src/state/selectors/taskSelectors.ts#L21)

I excluded completed projects because there was a scenario while I was doing QA when a pending task got completed due to the `completion_ends_project` logic. But it was still showing in the pending list. Wouldn't it be a more consistent logic if task shows in Completed list when the project is complete irrespective of whether task is active/pending/paused.

Reply:
Here is a screenshot of dashboard on main with a task T1 in pending list but completed project status. 

<img width="1429" alt="Screen Shot 2020-09-24 at 9 22 41 AM" src="https://user-images.githubusercontent.com/3481095/94172858-e49f4e00-fe47-11ea-94a1-a17c4467f2fd.png">

Here is a screenshot of dashboard for the same user. Here that task T1 shows up in the Completed tasks list.

<img width="1422" alt="Screen Shot 2020-09-24 at 9 23 49 AM" src="https://user-images.githubusercontent.com/3481095/94172863-e5d07b00-fe47-11ea-9848-22c187577c28.png">

Sanity check: The number of total tasks on the dashboard are same in both cases.

Reply:
**QA for Paused list** 

If the step with `completion_ends_project` is completed, the project gets completed

Reply:
Thank you for patiently explaining/QAing this! :)